src/base/rss/rssdownloadrule.h

```diff
@@ -88,6 +88,9 @@ namespace Rss
         bool operator==(const DownloadRule &other) const;
 
     private:
+        bool matches(const QString &articleTitle, const QString &expression) const;
+
+    private:
```

Comment:
I just saw the discussion with Chocobo1. No, we shouldn't follow that rule. I think most new headers we create/modify/refactor don't follow that rule anyway.